### PR TITLE
Torch blog

### PR DESCRIPTION
**Pull request by DhairyaLGandhi**

### CODE REVIEW

File: _posts/2020-06-03-using-Torch-with-Flux-2.md [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+
+In that spirit, we introduce [Torch.jl](https://github.com/FluxML/Torch.jl), a package that wraps the optimised kernels written for torch and PyTorch, and makes them available to use through Julia. We expect to do this to gain coverage over our existing GPU stack, and bring us plenty of performance improvements on GPUs along the way. As an example we take a popular object detection model - ResNet50 - and compare it with our existing tooling.
+
+![img](https://user-images.githubusercontent.com/20618666/79492160-e60a5600-801f-11ea-9d7f-dd04af243e44.png)
+*Image by github user https://github.com/gartangh*
+
+Not too shabby, we see about 2 orders of magnitude worth of improvement in runtime over the native kernels (all packages make use of the optimised cudnn kernels where possible). We however, see some areas of pending improvement. One of them being memory management. This is one of the areas which we intend to improve upon, a source of majority of overhead. 
+
```
Comment (ViralBShah) [inline _posts/2020-06-03-using-Torch-with-Flux-2.md:11]: Can we list all the models that will benefit from this work  -directly or indirectly?

File: _posts/2020-06-03-using-Torch-with-Flux-2.md [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+julia> using Flux, Metalhead, Torch
+
+julia> using Torch: torch
+
+julia> resnet = ResNet() # from Metalhead
+ResNet()
+
+julia> tresnet = resnet |> torch
+ResNet()
+```
```
Comment (ViralBShah) [inline _posts/2020-06-03-using-Torch-with-Flux-2.md:60]: Mention VGG, GoogleNet, DenseNet etc. are also available in MetalHead and link to the docs - say they will all benefit (if that is indeed the case).

File: _posts/2020-06-03-using-Torch-with-Flux-2.md [hunk 3]
```diff
@@ -0,0 +1,116 @@
...
+
+With [Flux.jl](https://github.com/Flux/Flux.jl), we have demonstrated how we see a high performance and flexible [differentiable programming](./2019-03-05-dp-vs-rl.md) framework to look like and show its intended use cases to be far reaching and allowing for meaningful speedups. We use it to target highly specialised hardware accelerators like [GPUs](https://fluxml.ai/Flux.jl/stable/gpu/) or [TPUs](https://arxiv.org/pdf/1810.09868.pdf). In our framework, we expect to use as much of the Julia ecosystem, and play nicely with custom types that our users define.
+
+In that spirit, we introduce [Torch.jl](https://github.com/FluxML/Torch.jl), a package that wraps the optimised kernels written for torch and PyTorch, and makes them available to use through Julia. We expect to do this to gain coverage over our existing GPU stack, and bring us plenty of performance improvements on GPUs along the way. As an example we take a popular object detection model - ResNet50 - and compare it with our existing tooling.
+
+![img](https://user-images.githubusercontent.com/20618666/79492160-e60a5600-801f-11ea-9d7f-dd04af243e44.png)
+*Image by github user https://github.com/gartangh*
```
Comment (ViralBShah) [inline _posts/2020-06-03-using-Torch-with-Flux-2.md:8]: I think min/median/etc. is not worthwhile to show. Show the best running times on all, across 3-4 different benchmarks.

File: _posts/2020-06-03-using-Torch-with-Flux-2.md [hunk 4]
```diff
@@ -0,0 +1,116 @@
+# Using Torch kernels inside Flux.jl
```
Comment (ViralBShah) [inline _posts/2020-06-03-using-Torch-with-Flux-2.md:1]: Suggested title: Flux.jl accelerated up to 100x with Torch kernels

File: _posts/2020-06-03-using-Torch-with-Flux-2.md (hunk 2 above)
Comment (DhairyaLGandhi) [reply _posts/2020-06-03-using-Torch-with-Flux-2.md:80]: Done

**Comment by ViralBShah** [comment]
@MikeInnes Feel free to edit the post directly. I just put in my edits in the latest commit.

**Pull request merged**